champions/src/main/java/me/mykindos/betterpvp/champions/weapons/impl/legendaries/scepter/MeridianBeam.java

```diff
@@ -0,0 +1,80 @@
+package me.mykindos.betterpvp.champions.weapons.impl.legendaries.scepter;
+
+import lombok.Getter;
+import me.mykindos.betterpvp.core.combat.events.CustomDamageEvent;
+import me.mykindos.betterpvp.core.utilities.UtilDamage;
+import me.mykindos.betterpvp.core.utilities.math.VectorLine;
+import me.mykindos.betterpvp.core.utilities.model.SoundEffect;
+import org.bukkit.Color;
+import org.bukkit.Location;
+import org.bukkit.Particle;
+import org.bukkit.Sound;
+import org.bukkit.entity.ArmorStand;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Player;
+import org.bukkit.event.entity.EntityDamageEvent;
+import org.bukkit.util.RayTraceResult;
+import org.bukkit.util.Vector;
+
+@Getter
+public class MeridianBeam extends ScepterProjectile {
+
+    public static final String NAME = "Meridian Beam";
+    private final double damage;
+
+    public MeridianBeam(Player caster, final Location location, double hitboxSize, double size, long expireTime, double damage) {
+        super(caster, hitboxSize, size, location, expireTime);
+        this.damage = damage;
+    }
+
+    @Override
+    protected void onTick() {
+        Location[] line = lastLocation == null  || lastLocation.equals(location)
+                ? new Location[] { location }
+                : VectorLine.withStepSize(lastLocation, location, 0.5).toLocations();
+
+        for (Location point : line) {
+            // Play travel particles
+            final Color color = Math.random() > 0.5 ? Color.fromRGB(184, 56, 207) : Color.fromRGB(174, 52, 179);
+            Particle.REDSTONE.builder()
+                    .location(point)
+                    .count(1)
+                    .extra(0.5)
+                    .data(new Particle.DustOptions(color, 1.3f))
+                    .receivers(60)
+                    .spawn();
+        }
+    }
+
+    @Override
+    public void redirect(Vector vector) {
+        super.redirect(vector);
+        new SoundEffect(Sound.ENTITY_SHULKER_BULLET_HIT, 0f, 1f).play(location);
```

Comment:
pitch of 0?

Reply:
sounded good in game

Reply:
Yes, pitch of 0

Reply:
Thats because theres another line under it which plays the same sound effect with a normal pitch value

Reply:
Both of them combine to make a more unique sound, that's on purpose

Reply:
they are overlayed to sound good together